### PR TITLE
Check if terrain simplified mesh is empty

### PR DESCRIPTION
**Pull request by baothientran**
Sometime, when the decimation is more aggressive, the simplified mesh will be empty. If it is, we will use uniform grid mesh instead. That should not happen too often and only when uniform grid mesh is too small

### CODE REVIEW

File: CDBTo3DTiles/src/CDBTo3DTiles.cpp [hunk 1]
```diff
@@ -431,9 +435,6 @@ Texture Converter::Impl::createImageryTexture(CDBImagery &imagery,
 
     Texture texture;
     texture.uri = textureRelativePath;
-    texture.magFilter = TextureFilter::LINEAR;
-    texture.minFilter = TextureFilter::LINEAR_MIPMAP_NEAREST;
```
Comment (lilleyse) [inline CDBTo3DTiles/src/CDBTo3DTiles.cpp:435]: Is this change supposed to be included?

**Comment by lilleyse** [comment]
Make sure to update `CHANGES.md` after https://github.com/CesiumGS/cdb-to-3dtiles/pull/23 is merged.

**Comment by lilleyse** [comment]
Is it possible to add a unit test for this?

**Comment by baothientran** [comment]
@lilleyse I updated the test in the new commit

**Comment by lilleyse** [comment]
Thanks @baothientran. While the test ended up being way more work than the fix itself I think it'll be a good safeguard for both for this edge case and the uniform grid mesh in general.

**Comment by lilleyse** [comment]
Also, please update `CHANGES.md`, then we can get this merged.

**Comment by baothientran** [comment]
@lilleyse updated

**Comment by lilleyse** [comment]
Thanks Bao!
